invenio_rdm_records/resources/serializers/schemas.py

```diff
@@ -12,6 +12,15 @@
 class BaseSchema(Schema):
     """BaseSchema for serializers in JSON."""
 
+    def get_doi(self, obj):
```

Comment:
Why do we favor related identifiers instead of `pids`?  see https://inveniordm.docs.cern.ch/reference/metadata/#external-pids for the `pids` field vs identifiers https://inveniordm.docs.cern.ch/reference/metadata/#alternate-identifiers-0-n

I see two options:
- Use just the `pids.doi.identifier` field
- Use both, meaning if the value is in `pids.doi.identifier` use that one, otherwise fallback to the `identifiers` field
WDYT?

Reply:
It is a system field, but it is configurable per instance (i.e. `DOI_ENABLED=False`) so there might be instances that do not have DOIs. While this is a big issue (e.g. datacite serialization should not work, I actually wonder how gracefully we fail in that case - bug 🤔? ), serializers such as bibtex should still work?

Reply:
so regarding BibTex, DOI is not needed for serialization (it is considered an optional param)
Regarding the rest of your comments I'm a bit lost :sweat_smile: